brozzler/js-templates/try-login.js.j2

```diff
@@ -48,6 +53,39 @@ var __brzl_tryLogin = function() {
             return;
         }
     }
+     
+
```

Comment:
Please avoid unnecessary empty lines, one empty line is fine to separate code parts.